src/ssd/ssd-extents.c

```diff
@@ -104,7 +104,7 @@ ssd_extents_update(struct ssd *ssd)
 
 	struct theme *theme = view->server->theme;
 
-	int width = view->current.width;
+	int width = MAX(view->current.width, LAB_MIN_VIEW_WIDTH);
```

Comment:
Should we limit the size of extents?

I found today that `xterm` can be resized to 1x1, and that this change makes cursor position and view geometry out of sync while resizing like the video below:

https://github.com/labwc/labwc/assets/22029524/e26b6c5d-0686-4c08-8eaf-60c3c15d0c19

Reply:
Nice catch! I only tested with the test client which doesn't allow resizing.
Limiting the extents seems correct to me but something seems off with the offset when then enlarging the surface again.

I'll look into it later.